text/0000-export-to-oci.md

```diff
@@ -0,0 +1,213 @@
+- OCI annotations. See [RFC](https://github.com/buildpacks/rfcs/pull/196)
+- Cosign integration. See [RFC](https://github.com/buildpacks/rfcs/pull/195)
+
+# What it is
+[what-it-is]: #what-it-is
+
+Currently the *Exporter*  writes either in an OCI image registry or a docker daemon, the idea is to add the capability to write into disk in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format only when the `-daemon` flag is used as argument AND the feature is enable using a new flag `-layout` or the default environment variable `CNB_LAYOUT_DIR`.
+
+## Examples
+
+### Exporting using the environment variable
+
+```=shell
+> export CNB_LAYOUT_DIR=oci
+>  /cnb/lifecycle/exporter -daemon my-app-image
+> tree /oci
+.
+└── oci/
+    └── my-app-image/
+        ├── blobs/
+        │   └── sha256/
+        │       └── 01..
+        ├── index.json
+        └── oci-layout
+
+```
+
+### Exporting using the command line flag
+
+```=shell
+>  /cnb/lifecycle/exporter -daemon -layout oci my-app-image
+> tree /oci
+.
+└── oci/
+    └── my-app-image/
+        ├── blobs/
+        │   └── sha256/
+        │       └── 01..
+        ├── index.json
+        └── oci-layout
+
+```
+As we can see there is a new folder called `oci` and inside that folder we can find our application image in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format
+
+Attempts to use this feature when the `-daemon` flag is not used could be ignore or show some warnings messages.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+The lifecycle phases affected by this new behavior is: [Export](https://buildpacks.io/docs/concepts/components/lifecycle/export/)
+
+At high level view of the propose solution can be summarize with the following container diagram from the C4 model
+
+![](https://i.imgur.com/XhtvT9j.png)
+
+Notice that we are relying on the OCI format Specification to expose the data for `Platforms`
+
+The following new input is proposed to be added to this phase
+
+| Input             | Environment Variable  | Default Value            | Description
+|-------------------|-----------------------|--------------------------|----------------------
+| `<layout>`      |  `CNB_LAYOUT_DIR` | "" | The root directory where the OCI image will be written. The presence of a none empty value for this environment variable will enable the feature. |
+
+
+- When the exporter is executed WITH the flag `daemon` it will check for the presence of the flag `layout` or the environment variable `CNB_LAYOUT_DIR`
+- IF any of the values are present THEN
+  - It will create a folder name `<image>` located at the path defined `<layout>` or `CNB_LAYOUT_DIR`
+  - In parallel of exporting the Image to the Daemon it will save the final Image in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format in the directory created before. The image will be saved using **uncompressed** layers
```

Comment:
From 3/3 working group: it was suggested to save the layers compressed, since we'll always need to compress them (to get the digest) and it is more expensive to compress than decompress.